### ISSUE TITLE
Add connection authorization?

### ISSUE THREAD

**Issue by joshbowyer** [viveris/uMTP-Responder#47]
Currently umtp-responder is very permissive and allows connections without any authorization (if I am mistaken please correct me). Can authorization functionality be added similarly to how Android handles it, by requiring some sort of "Allow This Device To Connect" dialogue on the host?

**Comment by jfdelnero** [comment]
Yes this is something possible. You can already use the message queue to mount/umount mtp drives from the device. 

I may also add a specialized message queue command to disconnect/reconnect the mtp connection like Android does.

Anyway the autorization user interface on the device is out of the scope of the uMTP responder. It will just provide the message queue interface to unlock/lock the mtp drives.

**Comment by jfdelnero** [comment]
Duplicate of #32

**Comment by jfdelnero** [comment]
The unlock/lock feature was implemented in the v1.5.1.

**Comment by joshbowyer** [comment]
Any documentation on how to enable it and what the scope of it is?

**Comment by jfdelnero** [comment]
The readme and the configuration file was updated today. (The official release was done 1 hour ago).
